src/main/java/com/tradeshift/service/HelloWorldInterface.java

```diff
@@ -0,0 +1,6 @@
+package com.tradeshift.service;
+
+
+public interface HelloWorldInterface {
```

Comment:
Generally speaking, you usually don't name classes `SomethingInterface` in java. I would probably make this the `HelloWorldService` whereas the thing you call the `HelloWorldService` right now would become `SpringHelloWorldService` or whatever name suits the actual implementation best. `MessageBasedHelloWorldService` or `EnglishHelloWorldService` for example.